translator/conventions/opentelemetry.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conventions
+
+// OpenTelemetry Semantic Convention values for Resource attribute names.
+// See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md
+const (
+	AttributeServiceName      = "service.name"
+	AttributeServiceNamespace = "service.namespace"
+	AttributeServiceInstance  = "service.instance.id"
+	AttributeServiceVersion   = "service.version" // proposed
+	AttributeLibraryName      = "library.name"
+	AttributeLibraryLanguage  = "library.language"
+	AttributeLibraryVersion   = "library.version"
+	AttributeContainerName    = "container.name"
+	AttributeContainerImage   = "container.image.name"
+	AttributeContainerTag     = "container.image.tag"
+	AttributeK8sCluster       = "k8s.cluster.name"
+	AttributeK8sNamespace     = "k8s.namespace.name"
+	AttributeK8sPod           = "k8s.pod.name"
+	AttributeK8sDeployment    = "k8s.deployment.name"
+	AttributeHostHostname     = "host.hostname"
+	AttributeHostID           = "host.id"
+	AttributeHostName         = "host.name"
+	AttributeHostType         = "host.type"
+	AttributeCloudProvider    = "cloud.provider"
+	AttributeCloudAccount     = "cloud.account.id"
+	AttributeCloudRegion      = "cloud.region"
+	AttributeCloudZone        = "cloud.zone"
+)
+
+// OpenTelemetry Semantic Convention values for general Span attribute names.
+// See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md
+// Peer attributes currently defined in database conventions
+// See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-database.md
+const (
+	AttributeComponent    = "component"
+	AttributePeerAddress  = "peer.address"  // to be obsoleted
+	AttributePeerHost     = "peer.hostname" // to be obsoleted
+	AttributePeerIpv4     = "peer.ipv4"     // to be obsoleted
+	AttributePeerIpv6     = "peer.ipv6"     // to be obsoleted
+	AttributePeerPort     = "peer.port"     // to be obsoleted
+	AttributePeerService  = "peer.service"  // to be obsoleted
+	AttributeNetTransport = "net.transport" // pull request #349
+	TransportTypeTCP      = "IP.TCP"
```

Comment:
Where do Transport* convention come from? I cannot find them in OpenTelemetry.

Reply:
These transport types are part of pull request 349 in opentelemetry-specification

Reply:
Let's not include them. They may not be approved or may be changed during review. I think it is best to define them directly in your other PR so that you do not need to wait.